tool/src/js/conversions.rs

```diff
@@ -138,6 +138,8 @@ pub fn gen_value_js_to_rust<'env>(
                 .and_then(|current| borrowed_current_to_root.get(current))
             {
                 entries.entry(named).or_default().push(param_name_buf);
+            } else if lifetime == &ast::Lifetime::Static {
```

Comment:
issue: this only works for slices, not, e.g. `&'static OpaqueObject`

I think we should extend the existing lifetime infra to stash these objects somewhere, the infra already understands the need to stash buffers

Reply:
Though perhaps we should just be forbidding static refs to opaques? They're basically impossible to construct anyway (except in CrabBake-like situations and it seems fine to box)

Reply:
`&'static OpaqueObject`s already have a static lifetime (although as you said, it's not possible to create them). Strings and slices on the other hand are owned in JS and need to be converted to static borrows, which is what this code tries to do.

Reply:
Yeah, I understand that difference, I think unifying the code would be good though.

Reply:
Do you want to allow `OpaqueObject -> &'static OpaqueObject` transforms?

Reply:
Not particularly, no